core/src/main/java/tc/oc/pgm/flag/LegacyFlagBeamMatchModule.java

```diff
@@ -197,16 +198,16 @@ public void show(MatchPlayer player) {
     }
 
     private void spawn(Player player, FakeEntity entity) {
-      location().ifPresent(l -> entity.spawn(l).send(player));
+      location().ifPresent(l -> entity.spawn(l.add(0, 3, 0)).send(player));
```

Comment:
This seems hazardous given that `location.add(` modifies the underlying value. Please check if this is a concern here

Reply:
It isn't because it is always guaranteed to be a clone, regardless i'm cleaning up a bit of this to make it more explicit